scripts/build/termux_create_pacman_subpackages.sh

```diff
@@ -121,15 +121,43 @@ termux_create_pacman_subpackages() {
 		# Allow packages to create arbitrary control files.
 		#termux_step_create_subpkg_debscripts
 
-		# Create the actual .deb file:
-		TERMUX_SUBPKG_PACMAN_FILE=$TERMUX_OUTPUT_DIR/${SUB_PKG_NAME}${DEBUG}-${TERMUX_PKG_FULLVERSION}-${SUB_PKG_ARCH}.pkg.tar.xz
+		# Configuring the selection of a copress for a batch.
+		case $PKG_COMPRESS in
+			"gzip")
+				COMPRESS=(gzip -c -f -n)
```

Comment:
No unnecessary global variables. Declare both COMPRESS and PKG_FORMAT with keyword "local".

Reply:
Ok